dashboard/test/models/regional_partner_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class RegionalPartnerTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Stream
+
+  test "regional partners initialized from tsv" do
+    RegionalPartner.find_or_create_all_from_tsv('test/fixtures/regional_partners.tsv')
+
+    assert_equal(2, RegionalPartner.count, 'test data contains 2 partners')
+    partner1 = RegionalPartner.find_by({name: 'A+ College Ready'})
```

Comment:
ruby style: you don't need the {}, or the () for that matter:

`RegionalPartner.find_by(name: 'A+ College Ready')` or even `RegionalPartner.find_by name: 'A+ College Ready'`